rbx_dom_lua/src/EncodedValue.lua

```diff
@@ -378,13 +378,26 @@ types = {
 			if pod == "Default" then
 				return nil
 			else
-				return PhysicalProperties.new(
-					pod.density,
-					pod.friction,
-					pod.elasticity,
-					pod.frictionWeight,
-					pod.elasticityWeight
-				)
+				-- Passing `nil` instead of not passing anything gives
+				-- different results, so we have to branch here.
```

Comment:
Thanks, I hate it

Reply:
Turns out `nil` becomes `0` when cast to a number. Love Roblox.